### PR TITLE
Make gem

### PR DESCRIPTION
**Pull request by b-artem**

### CODE REVIEW

File: lib/codebreaker_artem/cli.rb [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        guess
+      end
+
+      def show_hint(number = nil, position = nil)
+        return one_hint_only unless number && position
+        puts "HINT: Number #{number} is in position #{position + 1}"
+      end
+
+      def show_mark(mark)
+        puts mark
```
Comment (yamnich) [inline lib/codebreaker_artem/cli.rb]: no reason to use it here

File: lib/codebreaker_artem/game.rb [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+    def initial_values_set
+      @guess_count = 0
+      @score = 0
+      @hint_available = true
+    end
+
+    def generate_secret_code
+      @secret_code = ''
+      4.times { @secret_code << rand(1..6).to_s }
```
Comment (yamnich) [inline lib/codebreaker_artem/game.rb]: Array.new(4) { rand(1..6) }.join

File: lib/codebreaker_artem/starter.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    def self.play(game)
+      loop do
+        CLI.guess_prompt(game.guess_count)
+        case input = CLI.submit_guess
+        when nil then next
+        when :hint then CLI.show_hint(*game.hint)
+        else
+          mark = game.mark_guess(input)
+          CLI.show_mark(mark)
+          return CLI.win(input, game.score) if mark == '++++'
```
Comment (yamnich) [inline lib/codebreaker_artem/starter.rb]: mark == '++++' can we move it to method?

File: spec/codebreaker_artem/starter_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'spec_helper'
+
+RSpec.describe CodebreakerArtem::Starter do
+
```
Comment (yamnich) [inline spec/codebreaker_artem/starter_spec.rb]: can we remove it?

**Review by yamnich — APPROVED** [review]
Could you add CI please

File: lib/codebreaker_artem/cli.rb (hunk 1 above)
Comment (b-artem) [reply lib/codebreaker_artem/cli.rb]: Added some text for better output

File: lib/codebreaker_artem/game.rb (hunk 2 above)
Comment (b-artem) [reply lib/codebreaker_artem/game.rb]: Done

File: lib/codebreaker_artem/starter.rb (hunk 3 above)
Comment (b-artem) [reply lib/codebreaker_artem/starter.rb]: Moved to Validator.win_mark?

File: spec/codebreaker_artem/starter_spec.rb (hunk 4 above)
Comment (b-artem) [reply spec/codebreaker_artem/starter_spec.rb]: Done

**Pull request merged**